migrations/V158__Add_New_Model_Plan_Notification.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TYPE ACTIVITY_TYPE
+  ADD VALUE 'NEW_MODEL_PLAN' AFTER 'MODEL_PLAN_SHARED';
+
```

Comment:
It would be nice to have a comment on the column (the rest of the columns in this table have one, it makes the column documented in the migration as well as later in whatever db management tool you are using. Example below for the `daily_digest_complete` column
![image](https://github.com/CMSgov/mint-app/assets/99294987/4f5bd7aa-a317-4e91-82bf-09e892563a01)


```suggest
COMMENT ON COLUMN user_notification_preferences.new_model_plan IS 'Notification preference for when a model plan is created';

```